openmetadata-ui/src/main/resources/ui/src/pages/MyDataPage/MyDataPage.component.tsx

```diff
@@ -249,6 +285,26 @@ const MyDataPage = () => {
       });
   };
 
+  const fetchOMDMode = () => {
+    fetchSandboxConfig()
+      .then((res) => {
+        if (res.data) {
+          setIsSandbox(Boolean(res.data.sandboxModeEnabled));
+        } else {
+          throw '';
```

Comment:
Show unexpected response from the server.

Reply:
Same error is there as fallback text in catch block, so did not pass it explicitly with `throw`. Will add it.

Reply:
Addressed